giraffe/README.md

```diff
@@ -0,0 +1,341 @@
+
+  const lineLayer = {
+    type: "line",
+    x: "_time",
+    y: "_value",
+  }
+
+  const config = {
+    table: dataFromFlux.table,
+    layers: [lineLayer],
+  }
+
+  // ...
+
+  // return this element in your React rendering code:
+
+  &#60;div
+    style={{
+      width: "calc(70vw - 20px)",
+      height: "calc(70vh - 20px)",
+      margin: "40px",
+    }}
+  &#62;
+    &#60;Plot config={config} /&#62;
+  &#60;/div&#62;
+  </pre>
+
+## Config
+
+`<Plot>` requires a `config` prop which is an object with properties that serve three purposes:
+
+- customizing the [appearance](#appearance-properties) of `<Plot>`
+- specifying the [data](#data-properties) to be visualized
+- defining the columns and appearance of the [legend (tooltip)](#legend-tooltip-properties).
+
+<pre>
+  const config = {
+    <b><i>property</i></b>: <b><i>value</i></b>,
+    <b><i>...</i></b>
+  }
+</pre>
+
+### Appearance properties
+
+- **width**: _number. Optional._ The width in _CSS px_ of the Plot. Includes the space to the left and below the axes surrounding the ticks, tick labels, and axis labels. When not specified, the width of the parent element to `<Plot>` will determine.
+
+- **height**: _number. Optional._ The height in _CSS px_ of the Plot. Includes the space to the left and below the axes surrounding the ticks, tick labels, and axis labels. When not specified, the height of the parent element to `<Plot>` will determine.
+
+- **gridColor**: _string. Optional._ The _CSS color value_ of the grid lines. Applies to the inner horizontal and vertical rule lines. Excludes the axes and the border around the graph.
+
+- **gridOpacity**: _number. Optional. Recommendation: do not include. Defaults to 1 when excluded._ A value between 0 and 1 for the [_CanvasRenderingContext2D globalAlpha_](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalAlpha) of the grid lines. Applies to the inner horizontal and vertical rule lines. Excludes the axes and the border around the graph.
+
+- **showAxes**: _boolean. Optional. Recommendation: do not include. Defaults to true when excluded._ Indicates whether Plot axes should be visible. Applies to both x-axis and y-axis simultaneously.
+
+- **axisColor**: _string. Optional._ The _CSS color value_ of the axes and the border around the graph. Excludes the inner horizontal and vertical rule lines.
+
+- **axisOpacity**: _number. Optional. Recommendation: do not include. Defaults to 1 when excluded._ A value between 0 and 1 for the [_CanvasRenderingContext2D globalAlpha_](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalAlpha) of the axes and the border around the graph. Excludes the inner horizontal and vertical rule lines.
+
+- **xTicks**: _array[number, ...]. Optional._ An array of values representing tick marks on the x-axis. Actual data values and axis scaling may cause Plot to not render all of the given ticks, or Plot rendering may extend beyond all of the rendered ticks.
```

Comment:
Should we add a little blurb about how the ticks are calculated if none are passed in? Or what the default is?

Reply:
Yes. 🎉